ocaml/idl/datamodel_vm.ml

```diff
@@ -2093,34 +2093,8 @@ let t =
             "The host virtual hardware platform version the VM can run on"
         ; field ~qualifier:StaticRO
             ~lifecycle:[(Published, rel_dundee, "")]
-            ~doc_tags:[Windows]
-            ~default_value:
-              (Some
-                 (VCustom
-                    ( String.concat "\n"
-                        [
-                          "(try Rpc.Bool ("
-                        ; "let pool = List.hd \
-                           (Db_actions.DB_Action.Pool.get_all ~__context) in"
-                        ; "let restrictions = \
-                           Db_actions.DB_Action.Pool.get_restrictions \
-                           ~__context ~self:pool in "
-                        ; "let vendor_device_allowed = try List.assoc \
-                           \"restrict_pci_device_for_auto_update\" \
-                           restrictions = \"false\" with _ -> false in"
-                        ; "let policy_says_its_ok = not \
-                           (Db_actions.DB_Action.Pool.get_policy_no_vendor_device \
```

Comment:
I missed the use of `Pool.policy_no_vendor_device` when looking at this earlier. It is defined in datamodel_pool.ml as:
```
        ; field ~qualifier:RW ~in_product_since:rel_dundee
            ~default_value:(Some (VBool false)) ~ty:Bool
            "policy_no_vendor_device"
            "The pool-wide policy for clients on whether to use the vendor \
             device or not on newly created VMs. This field will also be \
             consulted if the 'has_vendor_device' field is not specified in \
             the VM.create call."
```
We need to check if this is still needed for some reason.

Reply:
It looks to me `policy_no_vendor_device` is not used outside this code except for tests. I do think we should try to remove that field as well because it is confusing. The problem is, that customers probably use it.